app/build.gradle

```diff
@@ -23,13 +25,31 @@ android {
     }
 }
 
+kapt {
+    correctErrorTypes = true
+}
+
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'androidx.appcompat:appcompat:1.0.2'
-    implementation 'androidx.core:core-ktx:1.0.2'
+    implementation 'androidx.appcompat:appcompat:1.1.0'
+    implementation 'androidx.core:core-ktx:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+
+    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0-beta01'
+
+    implementation 'com.airbnb.android:epoxy:3.4.2'
+    kapt "com.airbnb.android:epoxy-processor:3.4.2"
+
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
+
+    implementation 'com.github.bumptech.glide:glide:4.9.0'
+    kapt 'com.github.bumptech.glide:compiler:4.9.0'
+
+    implementation 'org.koin:koin-android:2.0.1'
+    implementation 'org.koin:koin-androidx-viewmodel:2.0.1'
+
     testImplementation 'junit:junit:4.12'
-    androidTestImplementation 'androidx.test:runner:1.1.1'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    androidTestImplementation 'androidx.test:runner:1.2.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
```

Comment:
Jovem, tem uma paradinha que a gente utiliza no projeto do picpay que é o controle de dependências através do Kotlin utilizando o `buildSrc`, da uma olhada nesses links aqui para ver se te ajuda, não precisa ja alterar nesse commit, pode fazer em uma branch a parte e subir ela a parte, so queria deixar esse insight :D 

https://handstandsam.com/2018/02/11/kotlin-buildsrc-for-better-gradle-dependency-management/
https://proandroiddev.com/migrate-to-gradle-kotlin-dsl-in-4-steps-f3e3b27e1f4d
https://medium.com/android-dev-br/usando-arquivos-kotlin-para-gerenciar-suas-dependência-b61d2480a501